javascript-modules-library/src/core/server/components/render/RenderChild.tsx

```diff
@@ -0,0 +1,29 @@
+import type { JSX } from "react";
+import { Render } from "./Render.js";
+import { useServerContext } from "../../hooks/useServerContext.js";
+import { AddContentButtons } from "../AddContentButtons.js";
+
+/**
+ * Renders a child of the current node, designated by its name.
+ *
+ * If the child node does not exist, it will display the "Add content" buttons.
+ */
+export function RenderChild({
+  name,
+  view,
+}: {
+  /**
+   * The name of the child node to render.
+   *
+   * In the CND file, it's what's after the `+` in the child node definition: `+ name (type)`.
+   */
+  name: string;
+  /** View to use when rendering the child. */
+  view?: string | undefined;
+}): JSX.Element {
+  const { currentNode } = useServerContext();
+  if (currentNode.hasNode(name)) {
```

Comment:
Should we log a warning if the child can't be retrieved?

Reply:
It's perfectly fine not to have a child, it shows `<AddContentButtons />` instead

Reply:
what I meant is if the `name` is set but that node does not exist